avm/ptn/deployment-script/import-image-to-acr/main.bicep

```
@@ -0,0 +1,255 @@
+metadata name = 'import-image-to-acr'
+metadata description = 'This modules deployes an image to an Azure Container Registry.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the deployment script resource.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+// ============ //
+// Parameters   //
+// ============ //
+
+@description('Required. The name of the Azure Container Registry.')
+param acrName string
+
+@description('Optional. How the deployment script should be forced to execute. Default is to force the script to deploy the image to run every time.')
+param forceUpdateTag string = utcNow()
+
+@description('Optional. Azure RoleId that are required for the DeploymentScript resource to import images. Default is AcrPush, which is needed to import into an ACR.')
```

```suggestion
@description('Optional. The RoleDefinitionId required for the DeploymentScript resource to import images. Default is AcrPush, which is needed to import into an ACR.')
```

Small update. However, may I suggest to not have this parameter but instead a boolean to control if the module should perform the role assignment (as opposed to the user needing to do this ahead of time). 
For one, the principal may not have the permissions to assign roles (so a control boolean might be helpful), while the roleDefinitionId will also be at least contributor.

---

Thanks for the fix of the typo - but was a bit more to my comment 😉

---

Oh yeah, reading. 🙄